en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,627 @@
+# Get Started with Choreo
+
+Choreo is a cloud-native engineering platform that allows you to develop integration solutions by designing APIs, API Proxies, webhooks, triggers, and scheduled tasks.
+
+This guide walks you through the following:
+
+- Developing, deploying, testing, and publishing a REST API. 
+
+- How an external application consumes the published REST API.
+
+- How to release new versions of the REST API with added functionality.
+
+Let's consider a use case where a developer named Jane designs an application for reading lists. A Choreo developer named John develops a REST API for this application to consume. The two developers will develop three versions of the solution in three iterations. Each new version will introduce new functionality as described below:
+
+
+- **Version 1**:
+
+    This version introduces the basic functionality where the application user can carry out the following activities for the reading list:
+
+     - Add books
+     - Update records of books
+     - Retrieve the complete reading list
+     - Delete books
+  
+       The information on the reading list will be removed from the system if you undeploy the REST API.
+
+- **Version 2**
+
+    This version involved connecting an external database to the REST API so that the reading list is persisted in the system memory even after the REST API is undeployed and redeployed.
+
+- **Version 3**
+
+    This version includes a front-end application that uses WSO2 Asgardeo as the identity provider. The front end-application allows different users to sign in with unique IDs and maintain separate reading lists.
```

Comment:
I would be great to mention the scenario first and then mention the technology behind it.

```
This version includes a front-end application that allows different users to sign in with unique login credetials and maintain separate reading lists. This is done using WSO2 Asgardeo as the identity provider.
```